views/js/index.php

```diff
@@ -23,9 +23,8 @@
 *  @license    http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
 *  International Registered Trademark & Property of PrestaShop SA
 */
-
```

Comment:
`100644 → 100755`
I dont know what is the rule for modules asset folders and files 😐 
@Quetzacoalt91 @PierreRambaud ?

Reply:
There is no official rule about file permission (except that 777 is forbidden)